prow/ANNOUNCEMENTS.md

```diff
@@ -25,9 +25,43 @@ Note: versions specified in these announcements may not include bug fixes made
 in more recent versions so it is recommended that the most recent versions are
 used when updating deployments.
 
+ - *Sep 27, 2018* If you are setting explicit decorate configs, the format has changed from
```

Comment:
I think this message is entirely too long? The only relevant part is needing to embed fields like timeout inside a `decoration_config` field.

Reply:
trimmed other fields.